Dockerfile

```diff
@@ -38,8 +38,7 @@ WORKDIR /app
 COPY ./src/frontend /app/
 
 RUN yarn install --frozen-lockfile && \
-    yarn build -o /dev/null && \
```

Comment:
What was the role of this build? Why it can be remove now?

Reply:
The role of this built was to generate the messages using babel-plugin-react-intl. We don't use this plugin at build time anymore so we can remove this build and extract messages using the CLI